### DISCUSSION TITLE
How to make DaisyUi work

### DISCUSSION THREAD

**Discussion by Stefanossq** [saadeghi/daisyui#3043] [General]
Hello everybody, i already installed the tailwindcss and daisyui and i can  make the commands to create a custom button, but, heres my problem: Example: i can <button class="btn btn-primary">Button</button>  but show that gray default button, how i make them equal to daisyui examples?
![seubotao](https://github.com/saadeghi/daisyui/assets/143559559/4d83203c-d48a-463a-bcd3-b362d597e985)

**Comment by MrVibe** [comment]
you maybe missing daisyui classes class="btn btn-primary"

**Reply by Stefanossq** [reply]
No , im already using  the daisyui classes class = 'btn btn-primary but only show that gray default

**Comment by saadeghi** [comment]
Please provide a reproduction link.

**Reply by Stefanossq** [reply]
![step1](https://github.com/saadeghi/daisyui/assets/143559559/b19919d0-6981-423e-87f4-59f51b5ffc11)
![step2](https://github.com/saadeghi/daisyui/assets/143559559/b2225e69-9c05-4f40-ac35-b94e0e0edfc4)
![step3](https://github.com/saadeghi/daisyui/assets/143559559/33963044-f44d-4ae8-9de0-968e27f8ef5e)

**Reply by Stefanossq** [reply]
Please help me

**Comment by Stefanossq** [comment]
OK but now all the button are white, i need change the colors pleaase help
![LOLLL](https://github.com/saadeghi/daisyui/assets/143559559/3e52a124-6ae5-4a53-b760-843f842078cf)

**Comment by saadeghi** [comment]
I asked for a reproduction link so I can run your project and see the result when I change the files. Screenshot is not helping. And it's not just about daisyUI. Whenever you're having an issue with any library, you should provide a minimum reproduction link so people can help you. You can use Stackblitz or Codesandbox or Tailwind Play to make a reproduction link.

**Reply by Stefanossq** [reply]
https://stackblitz.com/~/github.com/Stefanossq/geoquest

**Reply by saadeghi** [reply]
Remove the `themes: []` in `tailwind.config.js:11`. It disables all themes so there will be no colors without themes.
